activestorage/app/models/active_storage/blob/representable.rb

```diff
@@ -95,6 +95,26 @@ def representable?
   end
 
   private
+    def default_transformations
+      { format: default_variant_format }
+    end
```

Comment:
Since this is only used by `variant` (and not `preview`), would it be better to indicate that in the name, i.e. `default_variant_transformations`?

Reply:
My thinking was that these indirectly apply to previews, too, since `ActiveStorage::Preview` ends up calling `variant` on the generated preview image blob. But I do think it’s better as `default_variant_transformations`.